docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -279,6 +279,37 @@ throughout this style guide.
    Include visual of 'Do's and Don'ts!
 
 
+1.4 The Politics of Language
+----------------------------
+The language we choose to use is inherently political and we are aware of that fact. 
+This guide asserts that gender inclusive language is important and pushes for it's 
+inclusion. In doing so, we hope to advocate for gender minority inclusion 
```

Comment:
```suggestion
wider use in public health research and dissemination. In doing so, we hope to advocate for SGM inclusion
```
could use 'SGM' acronym since it was just defined in glossary above and we use it elsewhere in the guide